agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/LogDataMapper.java

```diff
@@ -19,114 +19,57 @@
  * DEALINGS IN THE SOFTWARE.
  */
 
-package com.microsoft.applicationinsights.agent.internal.exporter;
+package com.azure.monitor.opentelemetry.exporter.implementation;
 
-import com.azure.core.util.CoreUtils;
-import com.azure.monitor.opentelemetry.exporter.AzureMonitorTraceExporter;
 import com.azure.monitor.opentelemetry.exporter.implementation.builders.AbstractTelemetryBuilder;
 import com.azure.monitor.opentelemetry.exporter.implementation.builders.ExceptionTelemetryBuilder;
 import com.azure.monitor.opentelemetry.exporter.implementation.builders.Exceptions;
 import com.azure.monitor.opentelemetry.exporter.implementation.builders.MessageTelemetryBuilder;
-import com.azure.monitor.opentelemetry.exporter.implementation.logging.OperationLogger;
 import com.azure.monitor.opentelemetry.exporter.implementation.models.ContextTagKeys;
 import com.azure.monitor.opentelemetry.exporter.implementation.models.SeverityLevel;
+import com.azure.monitor.opentelemetry.exporter.implementation.models.TelemetryItem;
 import com.azure.monitor.opentelemetry.exporter.implementation.utils.FormattedTime;
 import com.azure.monitor.opentelemetry.exporter.implementation.utils.TelemetryUtil;
-import com.microsoft.applicationinsights.agent.internal.telemetry.TelemetryClient;
 import io.opentelemetry.api.common.AttributeKey;
 import io.opentelemetry.api.common.Attributes;
 import io.opentelemetry.api.trace.SpanId;
-import io.opentelemetry.sdk.common.CompletableResultCode;
 import io.opentelemetry.sdk.logs.data.LogData;
 import io.opentelemetry.sdk.logs.data.Severity;
-import io.opentelemetry.sdk.logs.export.LogExporter;
 import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
-import java.util.Collection;
+import java.util.function.Consumer;
 import javax.annotation.Nullable;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class LoggerExporter implements LogExporter {
+public class LogDataMapper {
 
-  private static final Logger logger = LoggerFactory.getLogger(LoggerExporter.class);
+  private static final Logger logger = LoggerFactory.getLogger(LogDataMapper.class);
 
   private static final AttributeKey<String> AI_OPERATION_NAME_KEY =
       AttributeKey.stringKey("applicationinsights.internal.operation_name");
 
-  private static final OperationLogger exportingLogLogger =
-      new OperationLogger(LoggerExporter.class, "Exporting log");
-
-  private final TelemetryClient telemetryClient;
   private final boolean captureLoggingLevelAsCustomDimension;
+  private final Consumer<AbstractTelemetryBuilder> defaultsPopulator;
```

Comment:
rename to defaultPopulator? is plural intended?

Reply:
ya, because it populates the defaults, I don't love the name either, but I'm ok with it for now unless u have another suggestion

Reply:
i used defaultPopulator..  how about just bulider? I can update mine if you aggree.

Reply:
names to be considered: 
`builder`, 
`defaultBuilder`,
`consumer`?

Reply:
how about `telemetryBuilderInit`? (because it initializes the telemetry builder)

or maybe `telemetryBuilderDefaults` or `telemetryDefaults`?

Reply:
telemetryDefault? singular please.

Reply:
renamed to `telemetryInitializer` as we discussed